libs/printing/src/render.tsx

```diff
@@ -177,7 +180,9 @@ export async function renderToPdf(
         path: outputPath,
         width: inchesToText(width),
         height: inchesToText(
-          height === Infinity ? Math.max(height, contentHeight) : height
+          height === Infinity
+            ? Math.max(PAPER_DIMENSIONS.Letter.height, contentHeight)
+            : height
```

Comment:
Maybe replace this with `Math.max(viewportHeight, contentHeight)` now that you've handled the `height === Infinity` case above when defining `viewportHeight`